core/src/main/scala/fs2/util/Task.scala

```diff
@@ -96,18 +96,21 @@ class Task[+A](val get: Future[Either[Throwable,A]]) {
       case a => Future.now(a)
     })
 
+  /** Returns a task that performs all evaluation asynchronously. */
+  def async(implicit S: Strategy): Task[A] = Task.start(this).flatMap(identity)
+
   /**
    * Run this `Task` and block until its result is available. This will
    * throw any exceptions generated by the `Task`. To return exceptions
-   * in an `\/`, use `attemptRun`.
+   * in an `Either`, use `attemptRun`.
    */
-  def run: A = get.run match {
+  def unsafeRun: A = get.run match {
     case Left(e) => throw e
     case Right(a) => a
   }
 
   /** Like `run`, but returns exceptions as values. */
-  def attemptRun: Either[Throwable,A] =
+  def unsafeAttemptRun: Either[Throwable,A] =
```

Comment:
Just curious.. why is this considered unsafe (for me unsafe meant "the type doesn't lie")?